src/css/main.scss

```diff
@@ -1350,3 +1350,20 @@ div.search-input:focus-within {
   }
 }
 
+// Save indicators styling
+
+.indicator {
```

Comment:
Let's name it a bit more descriptive. For example: `line-item-indicator` or even `line-item-save-indicator`